### PR TITLE
Add community wrapper blueprint for LLNL flux-framework

### PR DESCRIPTION
**Pull request by wkharold**
This PR adds a community "wrapper" blueprint for the LLNL [flux-framework](flux-framework.org)

The blueprint wraps the fluxfw-gcp module found [here](https://github.com/GoogleCloudPlatform/scientific-computing-examples/tree/main/fluxfw-gcp). It implements the approach described in [Flux support on the Cloud HPC Toolkit](https://docs.google.com/document/d/1QTsolSTvyXG_Xvr9geK_ZFAvhO6nQh5TmWRmcIND_9E/edit?usp=sharing&resourcekey=0-R3o3Se5THn5ohq-trDXwmA)

### CODE REVIEW

File: community/examples/flux-framework/flux-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+      subnetwork: $(flux-net.subnetwork_self_link)
+
+      service_account_emails:
+        manager: null
+        login: null
+        compute: null
+ 
+      cluster_storage:
+        mountpoint: ((module.homefs.network_storage.local_mount))
```
Comment (mr0re1) [inline community/examples/flux-framework/flux-cluster.yaml:52]: Please update to newly supported `$()` traversal functionality.
```go
- mountpoint: ((module.homefs.network_storage.local_mount))
+ mountpoint: $(homefs.network_storage.local_mount)
```

**Review by mr0re1 — CHANGES_REQUESTED** [review]
We don't accept PRs to `main` please issue a new one against `develop`.

**Comment by cboneti** [comment]
> We don't accept PRs to `main` please issue a new one against `develop`.

I just changed it to develop now.

**Comment by mr0re1** [comment]
PR validation failed with `Step #3 - "make-tests": addlicense failed`
Please add license headers to all new files, you can use other examples/READMEs as an example.

**Comment by cboneti** [comment]
`make add-google-license` will also automatically do that.

File: community/examples/flux-framework/flux-cluster.yaml (hunk 1 above)
Comment (wkharold) [reply community/examples/flux-framework/flux-cluster.yaml]: Done

**Comment by wkharold** [comment]
It looks like validation is failing on the newly supported `$()` traversal functionality.
When I tested it I had to rebuild `ghpc` on the develop branch to get it to parse the blueprint.

**Comment by mr0re1** [comment]
Could you please rebase on develop HEAD and force push ?

**Comment by mr0re1** [comment]
NOTE for reviewer: example was tried and all flux invocations worked as expected.

**Review by mr0re1 — APPROVED** [review]

**Pull request merged**